js/kraken.js

```diff
@@ -91,90 +92,6 @@ module.exports = class kraken extends Exchange {
                 'doc': 'https://www.kraken.com/features/api',
                 'fees': 'https://www.kraken.com/en-us/features/fee-schedule',
             },
-            'fees': {
-                'trading': {
```

Comment:
Let's keep the `fees['trading']` subsection, cause it's still correct. Let's just remove the funding fees, cause those are obsolete.